wgpu-core/src/command/clear.rs

```diff
@@ -157,6 +160,8 @@ impl Global {
             cmd_buf_raw.transition_buffers(dst_barrier.as_slice());
             cmd_buf_raw.clear_buffer(dst_raw, offset..end_offset);
         }
+
+        cmd_buf_data_guard.succeeded();
```

Comment:
nitpick: The name `succeeded` is an indicative phrase, which makes this seem like a query with no side effects. This isn't true, since it's used for arming an `Error` state in the underlying command encoder.

suggestion: Since this _does_ have side effects, maybe it'd be better to name it something like `mark_successful`?

Reply:
`mark_successful` sounds more appropriate. I will rename the fn.